com.warlordsoftwares.formatlab.yml

```diff
@@ -0,0 +1,169 @@
+      - /include
+      - /lib/pkgconfig
+      - /share/man
+    sources:
+      - type: git
+        url: https://github.com/LuaJIT/LuaJIT.git
+        #branch: v2.1
+        commit: 4c2441c16ce3c4e312aaefecc6d40c4fe21de97c
+
+  - name: mpv
+    buildsystem: simple
+    build-commands:
+      - python3 waf configure --disable-alsa --disable-build-date --disable-manpage-build
+        --prefix=/app
+      - python3 waf build
+      - python3 waf install
+    sources:
+      - type: archive
+        url: https://github.com/mpv-player/mpv/archive/v0.34.1.tar.gz
+        sha256: 32ded8c13b6398310fa27767378193dc1db6d78b006b70dbcbd3123a1445e746
+      - type: file
+        url: https://waf.io/waf-2.0.24
+        sha256: 93909bca823a675f9f40af7c65b24887c3a3c0efdf411ff1978ba827194bdeb0
+        dest-filename: waf
+    post-install:
+      - mkdir -p /app/etc/mpv/
+      # save screenshots at ~/Pictures/mpv-screenshots
+      - echo "screenshot-directory=~/Pictures/UMD4-MPV-screenshots" > /app/etc/mpv/mpv.conf
+    cleanup:
+      - /include
+      - /lib/pkgconfig
+      - /share
+
+  - name: ffmpeg
+    config-opts:
+      - '--disable-debug'
+      - '--disable-doc'
+      - '--enable-rpath'
+      - '--enable-gpl'
+      - '--disable-static'
+      - '--enable-shared'
+      - '--enable-libfontconfig'
+      - '--enable-libfreetype'
+      - '--enable-libopus'
+      - '--enable-libpulse'
+      - '--enable-librsvg'
+      - '--enable-libvpx'
+      - '--enable-libdav1d'
+      - '--enable-libxml2'
+      - '--enable-libx264'
+      - '--enable-libx265'
+      - '--enable-libv4l2'
+      - '--enable-libmp3lame'
+      - '--enable-libass'
+      - '--enable-gnutls'
+      - '--enable-libvorbis'
+      - '--enable-libaom'
+      - '--enable-libopenjpeg'
+    cleanup:
+      - /share/ffmpeg/examples
+    sources:
+      - type: archive
+        url: https://www.ffmpeg.org/releases/ffmpeg-4.4.1.tar.xz
+        sha256: eadbad9e9ab30b25f5520fbfde99fae4a92a1ae3c0257a8d68569a4651e30e02
+
+  - name: formatlab
+    buildsystem: simple
+    build-commands:
+      - install -Dm755 formatlab ${FLATPAK_DEST}/bin/formatlab
+      - install -Dm644 icon.png ${FLATPAK_DEST}/share/icons/hicolor/512x512/apps/com.warlordsoftwares.formatlab.png
+      - install -Dm644 formatlab.desktop ${FLATPAK_DEST}/share/applications/com.warlordsoftwares.formatlab.desktop
+      - desktop-file-edit --set-icon=${FLATPAK_ID} ${FLATPAK_DEST}/share/applications/com.warlordsoftwares.formatlab.desktop
+      - install -Dm644 com.warlordsoftwares.formatlab.metainfo.xml ${FLATPAK_DEST}/share/metainfo/com.warlordsoftwares.formatlab.metainfo.xml
+    sources:
+      - type: file
+        url: https://warlordsoftwares.com/media/apps_build/flatpak/formatlab
+        only-arches:
+          - x86_64
+        sha256: a08f07091c9c35da5eeeb72a5d9e9f3bc1c34f9506774effc9d4e5619bb2e985
+
+      - type: file
+        url: https://raw.githubusercontent.com/rishabh3354/FORMAT_LAB/main/snap/gui/icon.png
```

Comment:
Don't checkout git branches. this will randoly break the build.

Reply:
git raw file is static and won't change in future.

Reply:
this is false. This is the file at the HEAD of `main`. 

You should instead chekcout the repository since you get a few files from it.